biz.aQute.repository/src/aQute/bnd/repository/maven/pom/provider/PomConfiguration.java

```diff
@@ -62,4 +62,10 @@
 	 */
 	boolean transitive(boolean deflt);
 
+	/**
+	 * Sets the time in milliseconds when to check for changes in the pom-files
+	 */
+	// default: 5000ms
+	int pollTime(int pollTimeInSecs);
```

Comment:
Can we call this 

https://github.com/bndtools/bnd/blob/5d8500fd174eb7ec4228f383baba910d4d5573f8/biz.aQute.repository/src/aQute/bnd/repository/osgi/OSGiRepository.java#L67

Then we use the same property name `poll.time` as in other repos.